### ISSUE TITLE
Adjust "completely anonymous" wording on Web Discovery slide of onboarding - follow up to #39440

### ISSUE THREAD

**Issue by LaurenWags** [brave/brave-browser#39972]
### Description

Follow up to #39440 - found by @stephendonner 

On the Web Discovery slide of onboarding, the phrase "completely anonymous" is used. Per https://github.com/brave/brave-browser/issues/39440#issuecomment-2246412190 from @ShivanKaul this should be adjusted.

### Steps to reproduce

1. Clean profile
2. Go through onboarding until you get to the Web Discovery slide

### Actual result

see "completely anonymous" phrase used:

<img width="1208" alt="Screenshot 2024-07-24 at 2 18 07 PM" src="https://github.com/user-attachments/assets/4a49d260-dc92-44dc-b7c5-72a842a8dc38">

### Expected result

some other wording - TBD

### Reproduces how often

Easily reproduced

### Brave version (brave://version info)

Brave | 1.68.128 Chromium: 127.0.6533.73 (Official Build) (x86_64)
-- | --
Revision | 800676fd9ede2eb4174bac98c186e53749c7201f
OS | macOS Version 13.6.7 (Build 22G720)

### Channel information

- [X] release (stable)
- [X] beta
- [X] nightly

### Reproducibility

- [ ] with Brave Shields disabled
- [ ] with Brave Rewards disabled
- [ ] in the latest version of Chrome

### Miscellaneous information

cc @rebron @ShivanKaul 

Also cc @Uni-verse @hffvld @kjozwiak - I am not sure if this is applicable to Android/iOS, so I added those tags. If it is not, please feel free to remove them.

**Comment by kjozwiak** [comment]
Thanks for the heads up @LaurenWags. However, the above screen is not applicable for `iOS` and `Android`. Looks like @iccub already removed the `iOS` label and I removed the `Android` label. Here's the screens of our onboarding for both `iOS` & `Android` in case you're curious/for future reference:

**`Android onboarding`**

**`Example`** | **`Example`** | **`Example`**
----------------|----------------|-----------------
![Screenshot_20240801-131423](https://github.com/user-attachments/assets/f43a4aaa-c66f-45e8-a7fb-7fb0b374bd34) | ![Screenshot_20240801-131426](https://github.com/user-attachments/assets/32de9f8b-281d-4640-8841-c704f23368b4) | ![Screenshot_20240801-131431](https://github.com/user-attachments/assets/3755e9f0-c135-4ba6-b458-ee8612a396f7)

**`iOS onboarding`**

**`Example`** | **`Example`**
----------------|----------------
![IMG_0434](https://github.com/user-attachments/assets/52f34226-860d-4e78-801c-714d339ad4b1) | ![IMG_0435](https://github.com/user-attachments/assets/23372acd-fb88-4b7b-8508-867483061abc)

**Comment by LaurenWags** [comment]
thanks @kjozwiak!

@ShivanKaul can you look at the above iOS onboarding? There is mention of "completely private" which was removed on desktop via https://github.com/brave/brave-browser/issues/39440. Not sure if it needs to be removed from iOS also.

**Comment by ShivanKaul** [comment]
Yes it should be removed from iOS too, I thought my changes in https://github.com/brave/brave-core/pull/24415/files#diff-fdd06530d578a5df89b482cc03449f756be2972a0f07b2b620eacd67eb664c6d would change that too so this is surprising to me. @kjozwiak what version of iOS are those screenshots from? At least on the TestFlight browser I see the word "completely" being removed, which is correct.

**Comment by kjozwiak** [comment]
> Yes it should be removed from iOS too, I thought my changes in https://github.com/brave/brave-core/pull/24415/files#diff-fdd06530d578a5df89b482cc03449f756be2972a0f07b2b620eacd67eb664c6d would change that too so this is surprising to me. @kjozwiak what version of iOS are those screenshots from? At least on the TestFlight browser I see the word "completely" being removed, which is correct.

I don't remember which version I grabbed/used when posting the screenshots. Assuming it was probably `1.67.x` as `1.68.x` wasn't available/signed off yet. I checked `1.69.x` and `Completely` has been removed 👍 

![IMG_0455](https://github.com/user-attachments/assets/4d9f4227-a5f2-4fa4-bd08-0c3a11cfe9de)
